src/WebJobs.Extensions.DurableTask/AppServiceLinuxLoggerProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class AppServiceLinuxLoggerProvider : ILoggerProvider
+    {
+        public ILogger CreateLogger(string categoryName)
+        {
+            // TODO: I do not know if it's ok to throw away the categoryName
```

Comment:
I think what you are trying to with the passed category is to have different type of ILogger, based on our sync, you mentioned there would be one for storage, one for regular, and the other? Are they all going to the same kusto table or seperate? If latter, I think that would mean you need set the ConsolePrefix tio corresponding prefix that matches with category in your Logger class in order to have traces pumped into different kusto table.

Reply:
@wenhzha , I see what you mean! However, my thinking was that there would be only **two** types of loggers: one for Linux Consumption and another for Linux Dedicated. And that these loggers would figure out how to write the data from the three **sources** of logs: storage, durableTask, durableExtension.

In this case, they all go to the same table :) : DurableFunctionsEventLogs

Reply:
Although perhaps the right answer is that there would be three types of loggers/ ILoggers: one for Linux Consumption, one for Linux Dedicated, and another for Windows ETW events.

@cgillum , is that what you were thinking as well?

Reply:
@davidmrdavid The EtwEventSource class already takes care of getting events into the _DurableFunctionsEvents_ Kusto table. We don't need any `ILogger` help for that, so at most we'd want just two. I don't have a strong preference about whether consumption and dedicated are combined into a single ILogger or multiple ILoggers.